.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -74,17 +74,18 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
     - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
     - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
     - [ ] All JSX used for rendering exists in the render method
-    - [ ] The component has the minimum amount of code necessary for its purpose and it is
+    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
 - [ ] If a new CSS style is added I verified that:
-    - [ ] A similar style doesn’t already exist
-    - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
+    - [ ] A similar style doesn't already exist
+    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
 - [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
 - [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
-
+- [ ] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.
 
 <details>
-<summary><h4>PR Reviewer Checklist</h4></summary>
+<summary><h4>PR Reviewer Checklist to copy/paste into a new comment after the author checklist is completed, and to be completed by a Contributor+</h4></summary>
```

Comment:
you've update this to mention here that its to be completed by C+. I thought even we have to complete this when reviewing. Can you clarify? If its just C+ lets rename the section to be `C+ Reviewer Checklist` so there is no ambiguity. If its both let's rephrase to mention to be complete by all reviewers.

Reply:
Also this summary is so long that the dropdown comes out wierdly. 
![image](https://user-images.githubusercontent.com/14356145/177396998-2114db8e-77e3-40b0-9269-3a576fbdf3c6.png)

Can we split it up such that copy/paste onwards is on the next line

Reply:
1. The discussion concluded with what I understood: it's just the C+. However, it also highlighted that a PR Reviewer may not be a C+. As such, I think the current wording is correct.
2. Bump?